gobblin-modules/gobblin-kafka-common/src/main/java/org/apache/gobblin/kafka/writer/KafkaWriterConfigurationKeys.java

```diff
@@ -33,6 +36,14 @@
   static final String FAILURE_ALLOWANCE_PCT_CONFIG = "writer.kafka.failureAllowancePercentage";
   static final double FAILURE_ALLOWANCE_PCT_DEFAULT = 20.0;
 
+  public static final String WRITER_KAFKA_KEYED_CONFIG = "writer.kafka.keyed";
+  public static final boolean WRITER_KAFKA_KEYED_DEFAULT = false;
+  public static final String WRITER_KAFKA_KEYFIELD_CONFIG = "writer.kafka.keyField";
+  public static final String WRITER_KAFKA_KEYFIELD_DEFAULT = null;
+  public static final String WRITER_KAFKA_TYPEMAPPERCLASS_CONFIG = "writer.kafka.typeMapperClass";
+  public static final String WRITER_KAFKA_TYPEMAPPERCLASS_DEFAULT = AvroGenericRecordTypeMapper.class.getCanonicalName();
```

Comment:
Can use `getName()` in case this pattern gets copied and attached to a nested class since `getCanonicalName()` returns a name that does not work for instantiating nested classes.

Reply:
ack